src/game_message.cpp

```diff
@@ -250,6 +249,72 @@ static Game_Message::ParseParamResult ParseParamImpl(
 	return { iter, value };
 }
 
+Game_Message::ParseParamStringResult Game_Message::ParseStringParamImpl(
```

Comment:
Ignore ParseStringParam for now. This is just like ParseParam but it extracts a string, not a number. I need this for \x of the TextPlugin (the stuff in x is interpreted as a text identifier and inserted)
(needs testing)